oeps/processes/oep-0058-proc-translations-management.rst

```diff
@@ -0,0 +1,174 @@
+OEP-58: Translations Management
+###############################
+
+Using the GitHub Transifex App and a Separate Translation Repo as a Replacement for the edx-transifex-bot
+---------------------------------------------------------------------------------------------------------
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-58 <oep-0058-proc-translations-management>`
+   * - Title
+     - Translations Management
+   * - Last Modified
+     - 2022-08-10
+   * - Authors
+     - 
+       * Carlos Muniz <cmuniz@tcril.org>
+       * Feanil Patel <feanil@tcril.org>
+       * Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Ned Batchelder <ned@edx.org>
+   * - Status
+     - Under Review
+   * - Type
+     - Process
```

Comment:
I wrote this as a top level comment before but I'm not sure you saw - plus it's easier to reply to inline comments.

From OEP-1:

> A Process proposal describes a change to how the Open edX community functions.

> A Best Practice proposal describes a technology or implementation choice that the Open edX community believes all applicable Open edX services and/or libraries should use or follow.

> An Architecture proposal describes a concrete design problem with several potential solutions and the rationale behind the decision. The decision may be specific to a significant Open edX feature or a cross-cutting technical need.

You've specified this is a Process proposal, but those are typically... well, process documents - like Maintainers, DEPR, or "How OEPs Work". This feels like an architecture proposal.

Reply:
I've updated this 👍